client/packages/invoices/src/InboundShipment/DetailView/Footer/Footer.tsx

```diff
@@ -78,7 +78,7 @@ export const FooterComponent: FC<FooterComponentProps> = ({
       icon: <DeleteIcon />,
       onClick: onDelete,
       disabled: isDisabled || !isManuallyCreated,
-      disabledToastMessage: t('messages.cant-delete-generic'),
+      disabledToastMessage: t('messages.cant-delete-transferred'),
```

Comment:
Oh sorry Noel! I meant, when `isDisabled`, show the can't delete, and when `!isManuallyCreated`, show this one 😅